components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1192,7 +1192,7 @@ export class GitpodServerImpl<Client extends GitpodClient, Server extends Gitpod
     }
 
     async updateUserStorageResource(options: GitpodServer.UpdateUserStorageResourceOptions): Promise<void> {
-        const userId = this.checkAndBlockUser("updateUserStorageResource").id;
+        const userId = this.checkAndBlockUser("updateUserStorageResource", { uri: options.uri }).id;
```

Comment:
is there a way to log workspace (instance) id too?

Reply:
It is in `checkAndBlockUser`, also the `userId`